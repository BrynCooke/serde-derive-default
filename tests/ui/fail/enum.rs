use serde::Deserialize;
#[derive(Deserialize, serde_derive_default::Default)]
enum Test {
    Test(usize),

    TestStr(String),
}

fn default() -> usize {
    1
}

fn main() {}
