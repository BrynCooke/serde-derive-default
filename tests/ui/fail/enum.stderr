error: enums cannot derive Default
 --> tests/ui/fail/enum.rs:3:1
  |
3 | enum Test {
  | ^^^^
